### PR TITLE
Pom.xml updated

### PR DESCRIPTION
**Pull request by hamzaozturk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>plugins</artifactId>
         <groupId>org.igniterealtime.openfire</groupId>
-        <version>4.3.0-beta</version>
+        <version>4.4.0-beta</version>
```
Comment (guusdk) [inline pom.xml:7]: Why do we need a newer version of Openfire?

Comment (hamzaozturk) [reply pom.xml]: .factorypath file removed and version decreased.

**Comment by akrherz** [comment]
This closes #1 right?

**Comment by guusdk** [comment]
I think so, yes. @wrooot ?

**Comment by wrooot** [comment]
Yes, it closes #1

**Comment by hamzaozturk** [comment]
@guusdk anything waiting from me?

**Comment by guusdk** [comment]
Thanks!

**Pull request merged**